user/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+class User(models.Model):
+  account   = models.CharField(max_length   = 45)
+  password  = models.CharField(max_length   = 255)
+  name      = models.CharField(max_length   = 50)
+  email     = models.EmailField(max_length  = 45)
+  phone     = models.CharField(max_length   = 11)
+  create_at = models.DateField(auto_now_add = True)
+  update_at = models.DateField(auto_now     = True)
+
+  class Meta:
+    db_table = 'users'
+
+
+class Address(models.Model):
+  user      = models.ForeignKey('User', on_delete = models.CASCADE)
+  address   = models.CharField(max_length   = 100)
```

Comment:
나중에 ORM을 활용해서 주소를 조회할 때, Address.address 속성으로 접근해야된다. 가독성을 위해서 Address.name으로 접근할 수 있게 수정해주세요~
```suggestion
  name   = models.CharField(max_length   = 100)
```